dashboard/common/utils.py

```diff
@@ -89,6 +89,13 @@ def get_myla_globals(current_user):
         "logout": logout_url,
         "primary_ui_color": primary_ui_color,
         "google_analytics_id": google_analytics_id,
-        "help_url": help_url
+        "help_url": help_url,
```

Comment:
@chrisrrowland can you make sure the `help_url` is `target = _blank` as well, since it was not before. I am not sure if you are already addressing it here but that's not a requirement but would be nice to have it

Reply:
I'm not even sure where help_url is used :-o

Reply:
It AvatarModal component,
![Screen Shot 2020-08-14 at 12 39 37 PM](https://user-images.githubusercontent.com/8579775/90272387-53da5900-de2b-11ea-8d0a-64451ea7e741.png)

Reply:
Thanks, I'll include that as well.